ng-meetup/2017/05/angular-universal/index.html

```diff
@@ -154,9 +179,31 @@
           </code></pre>
         </section>
         <section>
+          <p>Don't manipulate the <code>nativeElement</code> directly</p>
+          <div class="fragment">
+            <p>Use the <code>Renderer</code>. This ensures that in any environment we're able to change our view.</p>
+            <pre><code>
+constructor(element: ElementRef, renderer: Renderer) {
+  renderer.setElementStyle(element.nativeElement, 'font-size', 'x-large');
+}
+            </code></pre></div>
+        </section>
+        <section>
+          <h3>Preboot</h3>
+          <ul>
+            <li class="fragment">Record and playback events - Which events and how they are replayed are configurable</li>
+            <li class="fragment">Respond immediately to events</li>
+            <li class="fragment">Maintain focus even if page is re-rendered</li>
+            <li class="fragment">Buffer client-side re-rendering for smoother transition</li>
+            <li class="fragment">Freeze page until bootstrap completes if user clicks button</li>
+          </ul>
+        </section>
+        <section>
           <h4>References</h4>
           <ul>
-            <li></li>
+            <li><a href="https://universal.angular.io/">Angular Universal</a></li>
+            <li><a href="https://docs.google.com/document/d/1q6g9UlmEZDXgrkY88AJZ6MUrUxcnwhBGS0EXbVlYicY/edit#heading=h.442ppinnqx9a">Angular Universal design doc</a></li>
+            <li><a href="https://github.com/angular/preboot">Preboot</a></li>
```

Comment:
Add the link to the working demo, too:
`<li><a href="https://github.com/FrozenPandaz/ng-universal-demo">https://github.com/FrozenPandaz/ng-universal-demo<a/></li>`